react/features/conference/middleware.js

```diff
@@ -80,3 +81,47 @@ StateListenerRegistry.register(
             }
         }
     });
+
+/**
+ * Configures the UI. In reduced UI mode some components will
+ * be hidden if there is no space to render them.
+ *
+ * @param {Store} store - The redux store in which the specified {@code action}
+ * is being dispatched.
+ * @private
+ * @returns {void}
+ */
+function _setReducedUI({ dispatch, getState }) {
+    const { reducedUI } = getState()['features/base/responsive-ui'];
+
+    dispatch(setToolboxEnabled(!reducedUI));
+    dispatch(setFilmstripEnabled(!reducedUI));
+}
+
+/**
+ * Does extra sync up on properties that may need to be updated after the
+ * conference was joined.
+ *
+ * @param {Store} store - The redux store in which the specified {@code action}
+ * is being dispatched.
+ * @private
+ * @returns {void}
+ */
+function _conferenceJoined({ dispatch, getState }) {
+    const { firstJoin } = getState()['features/conference'];
```

Comment:
This approach will not work on mobile because, unlike web, the state is not cleaned after each call, as the app keeps running.

A StateListenerRegistry which cleans it up after a meeting has ended would suffice I think.

Reply:
Turned to checking the type of the room.